Gui/opensim/utils/src/org/opensim/swingui/FileTextFieldAndChooser.java

```diff
@@ -84,6 +84,10 @@ public void setExtensionsAndDescription(String extensions, String description) {
    public void setFileFilter(FileFilter filter) {
       this.filter = filter;
    }
+	
+   public void setDialogTitle(String title) {
+	  this.title = title;   
+   }
 
```

Comment:
Are there scenarios where title wouldn't be cleared from one invocation to the next?

Reply:
I have tested and it always sets the new title. I don't think there is any scenario where it couldn't be cleared.